web-app/src/app/_services/lap.service.ts

```diff
@@ -45,8 +45,6 @@ export class LapService {
     newData.batterySecondsRemaining = data.batterysecondsremaining;
     newData.averageSpeed = this.rdService.getRoundedValue(data.averagespeed, 2);
 
-    newData.label = this.saveLabel();
```

Comment:
Would it be better to comment this out instead of entirely removing it, as it still exists in other branches? Just worried this line might be forgotten later

Reply:
The feature this was originally from is #141, so @Reddragoneer1 should watch out for it!

I think it's safe to get rid of for now, since it's only relevant for the one branch.